tests/network-tests/src/flows/proposals/cancellingProposal.ts

```diff
@@ -20,11 +21,11 @@ export default async function cancellingProposals({ api, query, lock }: FlowProp
   const createProposalFixture = new CreateProposalsFixture(api, query, [
     {
       type: 'Signal',
-      details: 'Proposal to cancel',
+      details: ('Proposal to cancel' as unknown) as Bytes,
       asMember: memberId,
       title: 'Proposal to cancel',
       description: 'Proposal to cancel',
-    },
+    } as ProposalCreationParams<'Signal'>,
```

Comment:
This assertion doesn't seem to be needed either

Reply:
Addressed in [b552fba](https://github.com/Joystream/joystream/pull/3997/commits/b552fba6a40a75129fb358c4b771150313077472)